From: Shiji Yang <yangshiji66@outlook.com>
Date: Fri, 9 May 2025 02:34:14 +0800
Subject: [PATCH] fix missing-prototypes warning

Fix the build warning on 6.12 kernel:

/home/db/owrt/build_dir/target-mips_24kc_musl/linux-lantiq_xrx200/ltq-ptm-vr9/ltq-ptm/ifxmips_ptm_vdsl.c:64:15: error: no previous prototype for 'cgu_get_pp32_clock' [-Werror=missing-prototypes]
   64 | unsigned long cgu_get_pp32_clock(void)
      |               ^~~~~~~~~~~~~~~~~~

Signed-off-by: Shiji Yang <yangshiji66@outlook.com>
---
 ifxmips_ptm_vdsl.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/ifxmips_ptm_vdsl.c
+++ b/ifxmips_ptm_vdsl.c
@@ -61,7 +61,7 @@ static void *g_xdata_addr = NULL;
 
 #define ENABLE_TMP_DBG                          0
 
-unsigned long cgu_get_pp32_clock(void)
+static unsigned long cgu_get_pp32_clock(void)
 {
 	struct clk *c = clk_get_ppe();
 	unsigned long rate = clk_get_rate(c);
